.github/workflows/build-and-push-rag-content.yaml

```diff
@@ -0,0 +1,106 @@
+name: "Build and push rag-content container image"
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+  schedule:
+    - cron: '15 02 * * *'
+
+env:
+  EXTRA_BUILD_ARGS: ""
+  IMAGE_NAME: quay.io/openstack-lightspeed/${{ github.event.repository.name }}
+
+jobs:
+  build-and-push-rag-content:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Free disk space (Ubuntu)
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          # This might remove tools that are actually needed,
+          # if set to "true" but frees about 6 GB.
+          tool-cache: true
+
+      - name: Checkout to the repository
+        uses: actions/checkout@v4
+
+      - name: Install podman
+        run: |
+          sudo apt update
+          sudo apt install podman -y
```

Comment:
Podman should be installed by default.  https://github.com/redhat-actions/podman-login?tab=readme-ov-file#podman-login

Reply:
Perfect, thanks! Let me test this out.

Reply:
Removed:)